Druid/DRJump.class.st

```diff
@@ -21,6 +21,14 @@ DRJump >> acceptVisitor: aVisitor [
 	^ aVisitor visitJump: self
 ]
 
+{ #category : #converting }
+DRJump >> asBackJumpLoop [
+
+	^ self controlFlowGraph allLoops
```

Comment:
strange too

Reply:
Hahah what? The name or the logic? I listen for suggestions :P